build/common.props

```diff
@@ -8,10 +8,25 @@
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)Key.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
```

Comment:
Let me test to make sure floating versions still work, but yeah I'm fine with that. Saves adding PackageReference in every csproj in the solution